components/Avtar.html

```diff
@@ -52,8 +52,8 @@ <h4>Example of Image Avatar</h2>
                 <div id="images">
                     <img  id="avatar-lg" src="Avtar1.jpg",alt="Italian Trulli">
                     <img  id="avatar-md" src="Avtar1.jpg",alt="Italian Trulli">
-                    <img  id="avatar-sm1" src="Avtar1.jpg",alt="Italian Trulli">
-                    <img  id="avatar-sm2" src="Avtar1.jpg",alt="Italian Trulli">
+                    <img  id="avatar-sm" src="Avtar1.jpg",alt="Italian Trulli">
```

Comment:
spelling correction is there .write `Avatar1.jpg` instead of  `Avtar1.jpg`

Reply:
This is a file name from where I can change it Avtar1.jpg into Avatar1.jpg.